analyzer/queries/queries.go

```diff
@@ -150,11 +150,17 @@ var (
       SELECT COALESCE(max(height), -1) as height
       FROM analysis.processed_blocks
       WHERE analyzer = $1
+    ),
+    gaps AS (
+      SELECT h
+      FROM highest_encountered_block, generate_series(GREATEST(1, $2::bigint), LEAST(highest_encountered_block.height, $3::bigint)) AS h
+      EXCEPT
```

Comment:
:information_source: EXCEPT them, whereas previously it fully relied on ON CONFLICT DO NOTHING